deps/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/deps/auth/IotHubSSLContextTest.java

```diff
@@ -67,11 +67,11 @@ private void generateSSLContextExpectations() throws NoSuchAlgorithmException, K
         new NonStrictExpectations()
         {
             {
-                mockedSSLContext.getInstance(anyString);
+                SSLContext.getInstance(anyString);
```

Comment:
shouldn't you be deleting Mocked instances? Is it referenced elsewhere?

Reply:
the mocked ssl context is used in our tests (that is the class under test). Instance references to static members don't make sense, and that is what has been removed.

Reply:
The ssl context is not directly tested, but IotHubSslContext class, which holds a reference to ssl context. The mocked instance is used for setting expectations etc.

Reply:
Is it complaining about static method getInstance?

Reply:
Does line 71 make any sense after this change?

Reply:
> Is it complaining about static method getInstance?

Yes, it does, since invoking a static method on an instance does not make any sense.

> Does line 71 make any sense after this change?

You are right, it doesn't seem to help with the test expectation. Let me see if Mockit does anything special here, in that case my change will need to be replaced by a suppression instead.

Reply:
you may want to check same with other places. If this is redundant piece of code, we should refactor it.

Reply:
I actually take that back, what we are doing here is: call SSLContext.getInstance() -> which should return an SSL context instance, which we assign to as the mocked instance in L71. 
So yes, it does make sense.

Reply:
I am not sure how that's being verified though. Look at constructorCreatesSSLContext test for reference